server/api/search.go

```diff
@@ -0,0 +1,30 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/odpf/stencil/search"
+)
+
+func (api *API) Search(c *gin.Context) {
```

Comment:
This needs to be moved to grpc handler. Since this API doesn't involve any file upload and downloads....grpc-gateway can easily handle both grpc and http handlers for this.